react/features/status-bar/index.js

```diff
@@ -0,0 +1 @@
+export * from './components';
```

Comment:
I think I didn't "git rm"-ed the folder properly. The status-bar was killed off and the icons were moved into a features/filmstrip subdirectory to follow what native does.